src/cljs/athens/events/remote.cljs

```diff
@@ -222,6 +222,35 @@
       {:fx [[:dispatch [:remote/send-event! page-delete-event]]]})))
 
 
+(rf/reg-event-fx
+  :remote/page-reindex-left-sidebar
+  (fn [{db :db} _]
+    (let [last-seen-tx               (:remote/last-seen-tx db)
+          reindex-left-sidebar-event (common-events/build-page-reindex-left-sidebar last-seen-tx)]
+      (js/console.debug ":remote/page-reindex-left-sidebar: local?" (pr-str reindex-left-sidebar-event))
+      {:fx [[:dispatch [:remote/send-event! reindex-left-sidebar-event]]]})))
+
+
+(rf/reg-event-fx
+  :remote/page-add-shortcut
+  (fn [{db :db} [_ uid]]
+    (let [last-seen-tx       (:remote/last-seen-tx db)
+          add-shortcut-event (common-events/build-page-add-shortcut last-seen-tx uid)]
+      (js/console.debug ":remote/page-add-shortcut:" (pr-str add-shortcut-event))
+      {:fx [[:dispatch [:remote/send-event! add-shortcut-event]]
+            [:dispatch [:remote/page-reindex-left-sidebar]]]})))
```

Comment:
If `:remote/page-reindex-left-sidebar` is part of `:remote/page-add-shortcut` we should not need to dispatch reindex.
It would be better if reindex would be part of resolving.

Reply:
Also, we can dispatch multiple events using `dispatch-n` 
```
{:fx  [[:dispatch-n [[:remote/send-event! add-shortcut-event]
                     [:remote/page-reindex-left-sidebar]]]]}
```

Reply:
> Also, we can dispatch multiple events using `dispatch-n`
> 
> ```
> {:fx  [[:dispatch-n [[:remote/send-event! add-shortcut-event]
>                      [:remote/page-reindex-left-sidebar]]]]}
> ```

True that, though in this case it should be just 1 event that is dispatched.